src/install/kubernetes/helm/install-helm.mdx

```diff
@@ -0,0 +1,101 @@
+---
+componentType: default
+headingText: Install the Kubernetes integration
+---
+
+New Relic has a bunch of Helm charts for the different components, each with its own set of features for the platform. You can install these components separately, but we really recommend using the [`nri-bundle`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) chart. New Relic provides this chart, which acts as a wrapper or a meta-package for the individual charts.
```

Comment:
"A bunch" feels too familiar/casual here. Would it be accurate to say: "We recommend using the `nri-bundle` chart to integrate with Kubernetes but, New Relic has many Helm charts available for a variety of components if you'd rather install them separately." Or something like that? I think it's better to start off with the action and the golden path and then move on to the variety of options/

Reply:
Also, when you say "New Relic provides this chart, which acts as a wrapper" it feels a little vague. Did New Relic build this chart? Is this native to New Relic or is it something that belongs with K8s or Helm? I think a little more specificity and clarity would be helpful here.

Reply:
> Also, when you say "New Relic provides this chart, which acts as a wrapper" it feels a little vague. Did New Relic build this chart? Is this native to New Relic or is it something that belongs with K8s or Helm? I think a little more specificity and clarity would be helpful here.

This means that if you decide to install K8s with the `nri-bundle` chart, it will include the individual charts that you can see in the github link. As it says, New Relic provides this graph and it's a package that includes the rest of the graphs to make the installation easier. That's why we recommend installing it that way. I've changed it to this:
`New Relic provides this chart, which works as a wrapper or a meta-package containing the individual charts.`
Do you think it is clearer now?